test/execution_test.ts

```diff
@@ -207,7 +207,7 @@ describe('Execution', () => {
               }),
             },
             execute: async ([valid]) => {
-              return valid ? {foo: 'blah'} : {foo: 123};
+              return valid ? {Foo: 'blah'} : {Foo: 123};
```

Comment:
How come many of these are changing to use normalized-style keys?

Reply:
The tests relied on the fact that we did normalization of the object output?

Reply:
I guess the argument is that we should probably change the expected output, instead of changing the `execute` function for the tests.